src/UnitTests/Projection/PolymorphismTests.cs

```diff
@@ -0,0 +1,63 @@
+﻿namespace AutoMapper.UnitTests.Projection;
+
+public class InMemoryMapPolymorphic : AutoMapperSpecBase
```

Comment:
This needs to be an integration test in the dedicated folder.

Reply:
moved